client/src/assets/scss/topbar.scss

```diff
@@ -0,0 +1,35 @@
+@import "./foundation/settings";
+@import "~foundation-sites/scss/foundation";
+@import "./foundation/variables"; 
+@include foundation-everything;
+
+.top-bar {
+  display: flex;
+  background: $white;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.top-logo {
+  display: flex;
+  background: url(../images/hiker-trees.jpg);
+  height: 120px;
```

Comment:
May want to specify this as `max-height` so that it is more responsive to changes in screen size

Reply:
For some reason, the hiker never makes an appearance... Hm... we may ask them to take a hike, as we spent waaaay to long looking for them.